tests/models/test_bedrock.py

```diff
@@ -581,3 +581,39 @@ async def test_bedrock_multiple_documents_in_history(
     assert result.output == snapshot(
         'Based on the documents you\'ve shared, both Document 1.pdf and Document 2.pdf contain the text "Dummy PDF file". These appear to be placeholder or sample PDF documents rather than files with substantial content.'
     )
+
+
+async def test_bedrock_group_consecutive_tool_return_parts(bedrock_provider: BedrockProvider):
+    """
+    Test that consecutive ToolReturnPart objects are grouped into a single user message for Bedrock.
+    """
+    import datetime
+
+    from pydantic_ai.messages import ModelRequest, ToolReturnPart
+    from pydantic_ai.models.bedrock import BedrockConverseModel
```

Comment:
Can we move these imports to the top of the file please?

Reply:
Done